scripts/install.sh

```diff
@@ -2,7 +2,8 @@
 # Install essential packages from Apt
 apt-get update -y
 # Python dev packages
-apt-get install -y build-essential python python-dev python-setuptools python-pip
+apt-get install -y build-essential python python-dev python-pip
+apt-get install -y git
```

Comment:
Why do we need git here?

Reply:
Git is required by pip to install packages from git repositories like the one mentioned in [requirements/common.txt](https://github.com/fossasia/open-event-server/blob/e1a15b1629f59b114ffc4c2afadae0517e3ec378/requirements/common.txt#L46)

Reply:
:+1: